README.md

```diff
@@ -273,17 +273,17 @@ The following example creates the database `my_database` with no additional opti
 }
 ```
 
-### Packages
+### Additional Packages
 
 The packages helper simply enables you to define a list of packages to install. Due to the way that chef merges array attributes you should take care to define all packages at the same precedence level. We may in the future support hash lists to enable purges too.
 
-Add `config-driven-helper::packages` to enable it.
+Add `config-driven-helper::packages-additional` to enable it.
 
 The following example installs both git and java packages.
 
 ```json
 {
-  "packages": [ "git", "java" ]
+  "packages-additional": [ "git", "java" ]
```

Comment:
note the syntax change to hash means the values in the example should also be changed